ckanext/security/controllers.py

```diff
@@ -200,7 +200,8 @@ def request_reset(self):
             id = request.params.get('user')
 
             context = {'model': model,
-                       'user': c.user}
+                       'user': c.user,
+                       u'ignore_auth': True}
```

Comment:
So this is the reason why our password reset override doesn't work on later versions of CKAN. They made a change to make the user listing/lookup functions require a logged in user by default, and when we were trying to get user info to send in the password reset email it was failing with an `unauthorized` message. However, we can tell the lookup to not care if the user is logged in or not for this case by setting this `ignore_auth` property.

Reply:
This will work on earlier and later versions of CKAN now, but we also should provide the feature flag in case consumers want to keep the behaviour and messaging that CKAN core has when resetting the password.